### PR TITLE
feat: wip: add global query context - PLEASE DONT REVIEW YET

### PR DESCRIPTION
**Pull request by subirjolly**
Closes #

### CODE REVIEW

File: src/dataExplorer/components/DataExplorer.tsx [hunk 1]
```diff
@@ -31,7 +33,12 @@ const DataExplorer: FC = () => {
       <LimitChecker>
         <div className="data-explorer">
           <HoverTimeProvider>
-            <TimeMachine />
+            {isFlagEnabled('Subir') && (
```
Comment (genehynson) [inline src/dataExplorer/components/DataExplorer.tsx:36]: lol nice flag name

**Comment by genehynson** [comment]
fyi you can make this PR a github "draft PR" so folks know not to review it 😄

File: src/dataExplorer/components/DataExplorer.tsx (hunk 1 above)
Comment (subirjolly) [reply src/dataExplorer/components/DataExplorer.tsx:36]: my parents also thought the same 🤣

**Comment by Asalem1** [comment]
What’s up here?

**Comment by subirjolly** [comment]
> What’s up here?

Slow and steady work going on, in parallel(ish).

**Pull request closed without merging**